crowbar_framework/app/views/nodes/_form.html.haml

```diff
@@ -113,6 +113,35 @@
           %span.help-block
             = t('.availability_zone_hint')
 
+
+      .form-group
+      = f.label :disk_roles, t('.disk_roles')
+
+      %table.table.table-middle
+        %thead
+          %tr
+            %th.col-sm-3
+              = t(".disk_name")
+            %th.col-sm-2
+              = t(".disk_role")
+        %tbody
+          - (@node.crowbar_wall["claimed_disks"] || {}).each do |disk_name, disk|
+            %tr
+              %td
+                %span.form-control{ :disabled => "disabled" }
+                  = disk_name
+              %td
+                %span.form-control{ :disabled => "disabled" }
+                  = disk["owner"]
```

Comment:
By putting the string as is, then we don't use the nice name that you have put in the en.yml files. This will be very visible for DRBD ;-)